src/components/monitor/monitor.jsx

```diff
@@ -39,7 +39,7 @@ const modes = {
 const getCategoryColor = (theme, category) => {
     const colors = getColorsForTheme(theme);
     return {
-        background: colors[categoryColorMap[category]].primary,
+        background: colors[categoryColorMap[category]].colourPrimary,
```

Comment:
This is mixing `color` and `colour` on the same line and I think we should not do that.

Reply:
This is a side effect of the themes being passed directly into Blockly and therefore needing the u, and also being referenced here, where Scratch historically didn't use the u. I agree it's icky, but changing all of Scratch's code to use the u also seems icky :/